modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/odata/retriever/test/OrganizationODataRetrieverCustomFieldsTest.java

```diff
@@ -253,6 +253,80 @@ public void testGetOrganizationsFilterByCustomFieldWithEqualsAndDateTextType()
 		Assert.assertEquals(organization1, organizations.get(0));
 	}
 
+	@Test
+	public void testGetOrganizationsFilterByCustomFieldWithEqualsAndDoubleArrayKeywordType()
+		throws Exception {
+
+		ExpandoColumn expandoColumn = _addExpandoColumn(
+			_expandoTable, RandomTestUtil.randomString(),
+			ExpandoColumnConstants.DOUBLE_ARRAY,
+			ExpandoColumnConstants.INDEX_TYPE_KEYWORD);
+
+		double[] columnValue = {1.0, 2.0, 3.0};
```

Comment:
Is it necessary? I think we can simplify like this:

double columnValue = 1.0;

This applies for the recent tests

What do you think?

Reply:
And if you agree on sending the changes, you just need a commit ;) Thanks!

Reply:
You're right, it is unnecessary to create an array if just one entry is later used. I'll push a commit simplifying this.
I'll add you as a collaborator .

Reply:
I resolved this conversation because @ricardocousolr update the PR with this commit: https://github.com/liferay-tango/liferay-portal/pull/117/commits/d0bb870eef9c52c173b5f16f3fe848ef1717a0e1